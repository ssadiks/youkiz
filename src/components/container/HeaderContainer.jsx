import React from 'react';
import { Link } from 'react-router-dom';

const HeaderContainer = () => (
  <div>
    <ul>
      <li><Link to="/">Home</Link></li>
      <li><Link to="/videos">Videos</Link></li>
      <li><Link to="/topics">not found</Link></li>
    </ul>
  </div>
);

export default HeaderContainer;
