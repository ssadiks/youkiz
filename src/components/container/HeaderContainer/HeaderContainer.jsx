import React from 'react';
import { Link } from 'react-router-dom';
import LanguageContainer from '../LanguageContainer/LanguageContainer';

const HeaderContainer = () => (
  <div>
    <ul>
      <li><Link to="/">Home</Link></li>
      <li><Link to="/videos">Videos</Link></li>
      <li><Link to="/bo">Back office</Link></li>
      <li><Link to="/topics">not found</Link></li>
    </ul>
    <LanguageContainer />
  </div>
);

export default HeaderContainer;
