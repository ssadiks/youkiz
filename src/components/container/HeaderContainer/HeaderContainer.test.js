import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import {
  BrowserRouter as Router,
} from 'react-router-dom';
import Adapter from 'enzyme-adapter-react-16';
import HeaderContainer from './HeaderContainer';

configure({ adapter: new Adapter() });

describe('HeaderContainer', () => {
  it('renders with default props', () => {
    const wrapper = shallow(
      <Router>
        <HeaderContainer />
      </Router>
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
