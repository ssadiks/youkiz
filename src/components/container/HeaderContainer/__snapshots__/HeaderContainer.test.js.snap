// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContainer renders with default props 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "blank",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <HeaderContainer />
</Router>
`;
